src/main/resources/application.yaml

```diff
@@ -138,7 +138,8 @@ security:
       - "startAppeal"
       - "editAppeal"
       - "submitAppeal"
-
+    caseworker-ia-iacjudge:
```

Comment:
could you please add a functional test to cover the document generated when end appeal is done by a judge?